client/apollo/css/src/Form/TextArea/TextAreaCommon.scss

```
@@ -38,11 +38,25 @@
     pointer-events: none;
   }
 
+  &:focus,
+  &:hover {
+    border: 1px solid var(--input-text-area-border-color);
+    box-shadow: 0 0 0 1px var(--input-text-area-border-color);
+  }
+
   &:not(:disabled, &--error) {
     &:focus,
-    &:hover {
-      border: 1px solid var(--input-text-area-border-color);
-      box-shadow: 0 0 0 1px var(--input-text-area-border-color);
+    &:hover,
+    &:not(:placeholder-shown, :focus) {
+      font-size: var(--input-text-area-font-size);
+      font-weight: var(--input-text-area-font-weight);
+      color: var(--input-text-area-font-color);
+    }
+  }
+
+  &:not(:placeholder-shown) {
+    &:not(:focus) {
```

Ca marcherais ça ? (comme au dessus)

```suggestion
  &:not(:placeholder-shown, :focus) {
```

---

Ca marche.
Merci